phase1/amisha/activity_schema.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+from sqlalchemy import TEXT, BigInteger, Column, DateTime, Float, Integer, String
+
+
+def activity_schema(sample_chunk):
```

Comment:
you do not need to create a function for this 
create a static list 
all these lists will be in the same file name schema.py